src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java

```diff
@@ -9,11 +16,22 @@
  */
 class KfConsumerTest {
 
-  /**
-   * @todo #41:30m/DEV Consumer subscribe Test
-   */
   @Test
-  void testSubscribe() {
+  void testSubscribe() throws FileNotFoundException {
+    final Consumer<String, String> consumer =
+      new KfConsumer<>(
+        new KfConsumerSettings<String, String>(
+          new XMLDocument(
+            new File("src/test/resources/consumer.xml")
+          )
+        ).consumer()
+      );
+    assertDoesNotThrow(
+      () ->
+        consumer.subscribe(
```

Comment:
@h1alexbel maybe if the subscription was successful, we can return some kind of identifier?

Reply:
@l3r8yJ 
maybe, I still investigating the best fits in consumer methods: ```subscribe``` and ```dataized = fetch```
but internal implementation of ```subscribe``` returns ```void```

Reply:
@h1alexbel got it